docs/SPIR-V.rst

```diff
@@ -590,31 +590,39 @@ Firstly, under certain `SigPoints <https://github.com/Microsoft/DirectXShaderCom
-+----------------------+----------+--------------------------+-----------------------+
-| SV_GroupThreadID     | CSIn     | ``LocalInvocationId``    | N/A                   |
-+----------------------+----------+--------------------------+-----------------------+
-| SV_GroupIndex        | CSIn     | ``LocalInvocationIndex`` | N/A                   |
-+----------------------+----------+--------------------------+-----------------------+
++-------------------------+----------+--------------------------+-----------------------+
+| HLSL Semantic           | SigPoint | SPIR-V ``BuiltIn``       | SPIR-V Execution Mode |
++=========================+==========+==========================+=======================+
+|                         | VSOut    | ``Position``             | N/A                   |
+| SV_Position             +----------+--------------------------+-----------------------+
+|                         | PSIn     | ``FragCoord``            | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_VertexID             | VSIn     | ``VertexIndex``          | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_InstanceID           | VSIn     | ``InstanceIndex``        | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_Depth                | PSOut    | ``FragDepth``            | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_DepthGreaterEqual    | PSOut    | ``FragDepth``            | ``DepthGreater``      |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_DepthLessEqual       | PSOut    | ``FragDepth``            | ``DepthLess``         |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_DispatchThreadID     | CSIn     | ``GlobalInvocationId``   | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_GroupID              | CSIn     | ``WorkgroupId``          | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_GroupThreadID        | CSIn     | ``LocalInvocationId``    | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_GroupIndex           | CSIn     | ``LocalInvocationIndex`` | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_OutputControlPointID | HSIn     | ``InvocationId``         | N/A                   |
++-------------------------+----------+--------------------------+-----------------------+
+| SV_PrimitiveID          | HSIn     | ``PrimitiveId``          | N/A                   |
```

Comment:
`SV_PrimitiveID` can also appear as `PCIn`, right? So we need another entry here.